doc/classes/DisplayServer.xml

```diff
@@ -1178,11 +1178,20 @@
 		<method name="status_indicator_set_icon">
 			<return type="void" />
 			<param index="0" name="id" type="int" />
-			<param index="1" name="icon" type="Image" />
+			<param index="1" name="icon" type="Texture2D" />
 			<description>
 				Sets the application status indicator icon.
 			</description>
 		</method>
+		<method name="status_indicator_set_menu">
+			<return type="void" />
+			<param index="0" name="id" type="int" />
+			<param index="1" name="menu_rid" type="RID" />
+			<description>
+				Sets the application status indicator native popup menu. If menu is set, primary activation callback is not triggered.
```

Comment:
> Will it show up instantly after being set?

No, it will show on clicking the icon (any mouse button), without sending signal from the activation handler to the control (in case of macOS show the menu, in case of Windows it's done in the activation event handler). So it's just a shortcut for showing menu (and setting menu position) manually, just set the menu and the rest is handled automatically.

If I understand correctly, `org.freedesktop.StatusNotifierItem.Menu` do the same, if it's set it will automatically handle menu without sending activation event.

Reply:
At least on macOS, it's not possible to have menu and activation callback (all indications work like this, action is the same for any mouse button, I guess since noting is "minimized" to the tray ever). I guess on Windows, it might make sense to do it only on right click and keep other mouse buttons as a normal activation callback. No sure what's `org.freedesktop.StatusNotifierItem` behavior, it might show menu only on a specific mouse button.

Reply:
Changed Windows to only use right-click (since it seems to be expected behavior), and added a note to the documentation.